js/ui/status/payg.js

```diff
@@ -29,6 +29,9 @@ class PaygIndicator extends PanelMenu.SystemIndicator {
         });
         this.menu.addMenuItem(this._item);
 
+        // Account ID
+        this._initializeAccountIDMenu();
```

Comment:
Isn't this missing the changes from https://github.com/endlessm/gnome-shell/pull/632/

Reply:
It is, thanks